ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/statementofmeans/DependantMapper.java

```diff
@@ -66,19 +66,24 @@ public Dependant from(CCDDependant ccdDependant) {
 
         OtherDependants otherDependants = mapOtherDependant(ccdDependant.getOtherDependants());
 
-        return new Dependant(
-            children,
-            ccdDependant.getNumberOfMaintainedChildren(),
-            otherDependants
-        );
+        return Dependant.builder()
+            .children(children)
+            .numberOfMaintainedChildren(ccdDependant.getNumberOfMaintainedChildren())
+            .otherDependants(otherDependants)
+            .anyDisabledChildren(false) // TODO
```

Comment:
Just curious...Any reason for this?

Reply:
To highlight that CCD integration for these changes needs to be completed. We were told it was a separate task

Reply:
Agreed with @dharmendrak to raise a separate story (ROC-4481) for CCD integration and to remove these null/false TODO placeholders.